src/credentials/sso/ssoAccessTokenProvider.ts

```diff
@@ -12,12 +12,12 @@ import { getLogger } from '../../shared/logger'
 import { StartDeviceAuthorizationResponse } from 'aws-sdk/clients/ssooidc'
 
 const CLIENT_REGISTRATION_TYPE = 'public'
-const CLIENT_NAME = `aws-toolkit-vscode-${Date.now()}`
+const CLIENT_NAME = `aws-toolkit-vscode`
 // According to Spec 'SSO Login Token Flow' the grant type must be the following string
 const GRANT_TYPE = 'urn:ietf:params:oauth:grant-type:device_code'
 // Used to convert seconds to milliseconds
 const MILLISECONDS_PER_SECOND = 1000
-const BACKOFF_TIME = 5000
+const BACKOFF_DELAY_MINUTES = 5000
```

Comment:
I think you meant milliseconds

Reply:
Yes, milliseconds.  Not a day and a half.